components/Footer.js

```diff
@@ -1,85 +1,192 @@
 import IconTwitter from "./icons/Twitter"
 import IconGithub from "./icons/Github"
-import IconYoutube from "./icons/YouTube"
+import IconYoutubeGray from "./icons/YouTubeGray"
 import IconLinkedIn from "./icons/LinkedIn"
 import NewsletterSubscribe from "./NewsletterSubscribe"
 import Container from "./layout/Container"
 import Link from "next/link"
+import AsyncAPILogoLight from "./AsyncAPILogoLight"
+import IconSlack from "./icons/Slack"
+import IconTwitch from "./icons/Twitch"
 
 export default function Footer() {
   return (
-    <footer className="bg-gray-900 mt-12">
-      <div className="max-w-screen-xl mx-auto py-12 px-4 overflow-hidden sm:px-6 lg:px-8">
-        <nav className="-mx-5 -my-2 flex flex-wrap justify-center">
-          <ul className="flex flex-wrap justify-center">
-            <li className="px-5 py-2">
-              <Link href="/about">
-                <a className="text-base leading-6 text-gray-500 hover:text-white transition ease-in-out duration-300">About</a>
+    <footer className="bg-gray-900 mt-12 margin: 0 auto">
+      <div className="max-w-screen-xl mx-auto py-10 px-4 overflow-hidden sm:px-6 lg:px-8 divide-y-2 divide-gray-700">
+        <nav className="-mx-5 -my-2 p-5 mt-2 flex flex-wrap justify-between ">
```

Comment:
![image](https://user-images.githubusercontent.com/76521428/171044846-db5461d8-82fb-49dd-80d5-6e84a5621156.png)

Kindly avoid giving negative paddings and margins to section as you can see it overlaps the sections on the website.

Reply:
Heyy @akshatnema , Thank you very much for your reply.

> As you can see, the hover effect is not seen in the text when the mouse pointer is kept on the links of Social section.

Actually, @mcturco instructed me to do it in this way. So, lets wait to see what @mcturco has to say regarding this.

> - sections in the mobile view should be center-aligned to give a better view.
> - correct the size and view of the netlify logo in the footer in mobile view

These are really suggestions, I will make those changes.

> avoid giving negative paddings and margins to section as you can see it overlaps the sections on the website.

But I didnt include that (in fact, I do not know how to give negative paddings and margins😄). It was included, when I fetched/pulled from the upstream.

Thank you very much for your suggestions.

Thank you😄